packages/devtools_app/lib/src/screens/memory/shared/primitives/memory_timeline.dart

```diff
@@ -123,7 +123,7 @@ class MemoryTimeline {
     _eventSamples.add(
       EventSample.gcEvent(
         timestamp,
-        events: extensionEvents,
+        theEvents: extensionEvents,
```

Comment:
rename to events

Reply:
Then it will conflict with the field name